CHANGES.rst

```diff
@@ -1,6 +1,14 @@
 0.18.2 (unreleased)
 ===================
 
+flat_field
```

Comment:
Minor nit, but we like to keep the change entires in alphabetical order of step/pipeline name, so these should go below `cube_build` in the list.

Reply:
Ok I think I finally have the change log correct. I just edited it on GitHub

Reply:
Yep, looks good to me now.